src/components/GroupSystems/GroupImmutableSystems.js

```diff
@@ -100,7 +100,7 @@ const GroupImmutableSystems = ({ groupName, groupId, ...props }) => {
   ) => {
     const enhancedConfig = enhancedEdgeConfig(groupName.toString(), config);
     const defaultData = await defaultGetEntities(
-      items,
+      null,
```

Comment:
Do we want to send null? Is it necessary when we could just fix the Reference error by simply declaring `let items = []` before this function call?

Reply:
We do not have to pass anything. Because, this is not needed at all, rather it may result in non-immutable systems to appear.